packages/hadron-build/lib/target.js

```diff
@@ -762,6 +774,41 @@ class Target {
 
     return res[0];
   }
+
+  static getAssetsForVersion(dir, version) {
+    const configs = supportedDistributions
+      .map((distribution) => {
+        return supportedPlatforms.map((platformConfig) => {
+          return { ...platformConfig, distribution };
+        });
+      })
+      .flat();
+
+    const assets = configs
+      .map((config) => {
+        const target = new Target(dir, { ...config, version });
+        return {
+          config: { ...config, version: target.version, channel: target.channel },
+          assets: target.assets
+        };
+      })
+      .flat();
+
+    return assets;
+  }
+
+  static getChannelFromVersion(version) {
+    // extract channel from version string, e.g. `beta` for `1.3.5-beta.1`
+    const mtch = version.match(/-([a-z]+)(\.\d+)?$/);
+    if (mtch) {
+      return mtch[1].toLowerCase();
```

Comment:
is this a typo?
```suggestion
    const match = version.match(/-([a-z]+)(\.\d+)?$/);
    if (match) {
      return match[1].toLowerCase();
```

Reply:
I copy pasted it from old code without changes and I think someone was just really into unnecessary variable name shortening 😆 Will update!